moj-node/server/views/components/featured-content-item/template.njk

```diff
@@ -31,7 +31,7 @@
   game: "icon-link"
 }%}
 
-<a data-featured-id="{{ params.id }}" class="govuk-hub-featured-link" href="{{ params.contentUrl }}" {% if params.contentType === 'pdf' %} target="_blank" {% endif %}>
+<a data-featured-id="{{ params.id }}" class="govuk-hub-featured-link govuk-link" href="{{ params.contentUrl }}" {% if params.contentType === 'pdf' %} target="_blank" {% endif %}>
```

Comment:
can we swap the css round
as in `govuk-link` being the default and `govuk-hub-featured-link` being the override

Reply:
👍